package.json

```
@@ -82,6 +82,9 @@
     "release-it-lerna-changelog": "^2.3.0",
     "webpack": "^5.67.0"
   },
+  "peerDependencies": {
+    "ember-tether": "^2.0.1"
+  },
```

If it's optional then should a `peerDependenciesMeta` be included?

```suggestion
  "peerDependencies": {
    "ember-tether": "^2.0.1"
  },
  "peerDependenciesMeta": {
    "ember-tether": {
      "optional": true
    }
  },
```